ldpd/lde.c

```diff
@@ -1653,6 +1653,25 @@ lde_del_label_chunk(void *val)
 	free(val);
 }
 
+static int
+lde_release_label_chunk(struct listnode *node, uint32_t start, uint32_t end)
```

Comment:
It would be clearer (imo) if this just released the chunk by doing the zapi work, and taking a `struct label_chunk *` as its argument. As it is here, it deals with zebra, with the global list, and with the allocated memory.

Reply:
To deal with zebra, we don't need struct label_chunk * . What about changing it like,

static int
lde_release_label_chunk(uint32_t start, uint32_t end)
{
    int      ret;
    ret = lm_release_label_chunk(zclient_sync, start, end);
    if (ret < 0) {
        log_warnx("Error releasing label chunk!");
        return -1;
    }
    return (0);
}